<?xml version="1.0"?>

<launch>

  <!-- Parameters. -->
  <arg name="number_of_desired_fixes"                 default="5000"/>
  <arg name="height_base_station_from_ground"         default="0.0"/>

  <!-- Piksi. -->
  <include file="$(find piksi_multi_rtk_ros)/launch/piksi_multi_base_station.launch"/>

  <!-- Geodetic Survey. -->
  <node pkg="piksi_multi_rtk_ros" type="geodetic_survey" name="geodetic_survey" output="screen">
    <param name="number_of_desired_fixes"             value="$(arg number_of_desired_fixes)"/>
    <param name="height_base_station_from_ground"     value="$(arg height_base_station_from_ground)"/>
  </node>

</launch>
